sample_files/compare.expected

```diff
@@ -56,7 +56,7 @@ sample_files/dart_1.dart sample_files/dart_2.dart
 107d073c2922ff14e9ba66ee0f489592  -
 
 sample_files/devicetree_1.dts sample_files/devicetree_2.dts
-f7c4e7b4444b02d87b2eec1485d86211  -
```

Comment:
There is a small change in highlighting output, with one `/` character becoming bold in the new version in the sample file. This looks acceptable to me. The highlighting for this language remains very minimalistic both before and after the unvendoring.

Reply:
Fair enough. I use this file to see if anything has changed, but it's often minor things that are totally reasonable.